### PR TITLE
es_us spanish translation of papagayo GUI

### PR DESCRIPTION
**Pull request by ogallagher**
See linked comment for progress so far: https://github.com/morevnaproject-org/papagayo-ng/issues/111#issuecomment-1510651616

### CODE REVIEW

File: .gitignore
Comment (ogallagher) [inline .gitignore:1]: these are files related to mac and the vs code IDE

File: papagayo-ng.pro [hunk 1]
```diff
@@ -13,11 +13,13 @@ SOURCES += papagayo-ng.py \
         WaveformViewRewrite.py \
         SettingsQT.py \
 
-FORMS += rsrc\papagayo-ng2.ui \
-        rsrc\settings.ui \
-        rsrc\about_box.ui
+# backslash \ paths only work on Windows, but forwardslash / paths should work on both linux and win?
+FORMS += rsrc/papagayo-ng2.ui \
+        rsrc/settings.ui \
+        rsrc/about_box.ui
```
Comment (ogallagher) [inline papagayo-ng.pro:19]: I'd like to confirm this is true for Windows before merging.

**Comment by steveway** [comment]
I will merge this for now.
This seems to be a good improvement.
We can always fix stuff which is not finished or buggy.
Thanks for your work!

**Comment by ogallagher** [comment]
Sweet! Happy to help.

**Pull request merged**